siobhangpn/dinner.txt

```diff
@@ -0,0 +1,3 @@
+2017-09-24  Wagyu Cheese & Bacon Burger
```

Comment:
You’re uploading all files directly to the root of this project. Not to the `site/class-1-load` directory.